### DISCUSSION TITLE
Remove the initial home page?

### DISCUSSION THREAD

**Discussion by ghost** [fuma-nama/fumadocs#673] [Q&A]
How can you remove the "home page" thing with Fuma Docs? Basically so that the actual pages/documentation are not at /docs, but rather /

I found a similar discussion post but it contained nothing helpful. I've searched around the internet to no avail. 

All help is appreciated...

**Comment by fuma-nama** [comment]
Move the catch all route to app folder (basically how you change the routing of App Router apps), and update the base URL of pages in source.ts.

You may see the I18n example in this repo, similar implementation except with I18n enabled.
